### PR TITLE
Update MAINTAINER field

### PR DESCRIPTION
**Pull request by harmw**
Bert will be taking over 👍

### CODE REVIEW

**Comment by jbeich** [comment]
I can't verify you're the current maintainer and not a doppelganger. Ports use mail addresses to identify maintainers, not real names or accounts outside of `.freebsd.org` domains. Our Bugzilla installation automates such workflow but on Github... not sure how to do it without exposing mail to spammers.

File: net/cloud-init/Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ CATEGORIES=	net python
 MASTER_SITES=	http://launchpad.net/${PORTNAME}/trunk/${PORTVERSION}/+download/
 PKGNAMEPREFIX=	${PYTHON_PKGNAMEPREFIX}
 
-MAINTAINER=	harm@weites.com
+MAINTAINER=	bertjw@regeer.org
```
Comment (jbeich) [inline net/cloud-init/Makefile]: This address appears to be different from the one used in [bug 212453](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=212453). Is it still the same person?

**Comment by jbeich** [comment]
Also, do you have Bert's consent?

**Comment by harmw** [comment]
Yes I have, he actually offered to take over. I pinged him a day ago, referring to this PR. As you can see, my GH username doesn't differ much from the e-mail address being replaced 😅

**Pull request closed without merging**